cli/search/search.go

```diff
@@ -115,6 +119,60 @@ func GetVersionsFromGitRemote(repo string) ([]version.Version, error) {
 	return versions, nil
 }
 
+// GetCommitsFromGitLocal returns commits list from specified local git repo.
+func GetCommitsFromGitLocal(repo string) ([]string, error) {
+	commits := []string{}
+
+	_ = repo
+
+	if _, err := exec.LookPath("git"); err != nil {
+		return nil, fmt.Errorf("'git' is required for 'tt search' to work")
+	}
+
+	cmd := exec.Command("git", "log", "--oneline")
+	cmd.Dir = repo
+
+	output, _ := cmd.Output()
+
+	lines := strings.Split(strings.TrimSpace(string(output)), "\n")
+
+	// No tags found.
+	if len(lines) == 1 && lines[0] == "" {
+		return commits, nil
+	}
+
+	for _, line := range lines {
+		commits = append(commits, line[0:7])
```

Comment:
Let's reserve `commits` buffer space for the lengths of `lines`.

Reply:
fixed